nix/examples.sh

```diff
@@ -0,0 +1,4 @@
+dune build examples/add_commutative.py examples/definitions_ast.py examples/syntax_error.py
+dune exec -- python3 _build/default/examples/add_commutative.py
+dune exec -- python3 _build/default/examples/definitions_ast.py
+dune exec -- python3 _build/default/examples/syntax_error.py
```

Comment:
An easier way to do this is with a rule.

Reply:
this has been factored into a rule, marking resolved.